### ISSUE TITLE
Revise loss functions

### ISSUE THREAD

**Issue by Foster-ux** [XMU-Kuangnan-Fang-Team/GENetLib#144]
Hi GENetLib team! I found that the training in `scalarl2train` was inconsistent with that in `scalarmcpl2train` in terms of calculating the loss function. So I revised the code to make them consistent. Please check PR.
